src/user/room-invitation.ts

```diff
@@ -49,6 +51,18 @@ export class RoomInvitation extends Accessory implements Acceptable {
     return newRoomInvitation
   }
 
+  public static async serialize (invitation: RoomInvitation): Promise<string> {
```

Comment:
See: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify#Description
> If the value has a toJSON() method, it's responsible to define what data will be serialized.

Please follow the standard JavaScript design by renaming `serialize` to `toJSON()`

Reply:
sure, will do.